### PR TITLE
fix: avoid duplication of interfaces

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

**Comment by lpatiny** [comment]
In test cases I would like to see a sequence of 150 x `0` and a sequence 1, 2, 3 … to 150 and check that the result of the jcamp parser is the same for all the 4 compression techniques

It would be nice as well to have 1, 2, 3 … to 150 and -1 to -150 just to check the DIFDUP

File: src/shapes/1d/pseudoVoigt/PseudoVoigt.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ export const pseudoVoigtFwhmToWidth = (fwhm: number, mu = 0.5) => {
 };
 
 export const getPseudoVoigtArea = (options: GetPseudoVoigtAreaOptions) => {
-  const { fwhm, height = 1, mu = 0.5 } = options;
+  const { fwhm = 500, height = 1, mu = 0.5 } = options;
   if (fwhm === undefined) {
```
Comment (targos) [inline src/shapes/1d/pseudoVoigt/PseudoVoigt.ts:131]: this condition is now impossible

File: src/shapes/1d/lorentzian/Lorentzian.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ export const lorentzianFwhmToWidth = (fwhm: number) => {
 };
 
 export const getLorentzianArea = (options: GetLorentzianAreaOptions) => {
-  const { fwhm, height = 1 } = options;
+  const { fwhm = 500, height = 1 } = options;
```
Comment (targos) [inline src/shapes/1d/lorentzian/Lorentzian.ts:85]: same

File: src/shapes/1d/gaussian/Gaussian.ts [hunk 3]
```diff
@@ -109,7 +115,7 @@ export function gaussianFwhmToWidth(fwhm: number) {
 }
 
 export function getGaussianArea(options: GetGaussianAreaOptions) {
-  let { fwhm, sd, height = 1 } = options;
+  let { fwhm = 500, sd, height = 1 } = options;
 
   if (sd) fwhm = gaussianWidthToFWHM(2 * sd);
```
Comment (targos) [inline src/shapes/1d/gaussian/Gaussian.ts:121]: same on the line below

**Pull request merged**